lib/validator.js

```diff
@@ -509,6 +509,22 @@ Specberus.prototype.getFeedbackDueDate = function () {
     return dates;
 };
 
+// Return array of group names, e.g. ['Internationalization Working Group', 'Technical Architecture Group']
+Specberus.prototype.getDelivererNames = function () {
+    const sotd = this.getSotDSection();
+    const delivererNamesRegex =
+        /This document was (?:produced|published) by the (\w+ Working Group|\w+ Interest Group|Technical Architecture Group|Advisory Board)( and the (\w+ Working Group|\w+ Interest Group|Technical Architecture Group|Advisory Board))?/;
```

Comment:
The regex will not work if the group name contains whitespaces.
`\w` is equivalent to `[a-zA-Z0-9_]`